src/libraries/Microsoft.Extensions.FileSystemGlobbing/src/Microsoft.Extensions.FileSystemGlobbing.csproj

```diff
@@ -1,23 +1,19 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <RootNamespace>Microsoft.Extensions.FileSystemGlobbing</RootNamespace>
-    <AssemblyName>Microsoft.Extensions.FileSystemGlobbing</AssemblyName>
-    <Description>File system globbing to find files matching a specified pattern.</Description>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
     <NoWarn>$(NoWarn);CS1591</NoWarn>
-    <PackageTags>files;filesystem;glob;globbing</PackageTags>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <IsPackable>true</IsPackable>
-    <IsShipping>true</IsShipping>
   </PropertyGroup>
 
   <ItemGroup>
     <InternalsVisibleTo Include="Microsoft.Extensions.FileSystemGlobbing.Tests" />
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Include="$(SharedSourceRoot)HashCodeCombiner\*.cs" />
+    <Compile Include="$(CommonPath)Extensions\HashCodeCombiner\HashCodeCombiner.cs">
```

Comment:
We already have multiple "hashing helpers" code in the repo. Can we use one of the existing approaches instead of making yet another one?

[https://github.com/dotnet/runtime/blob/327801e8b838b97950cb8daaeef724acbe4e3c5f/src/libraries/System.Private.CoreLib/src/System/Numerics/Hashing/HashHelpers.cs#L7-L11](https://github.com/dotnet/runtime/blob/327801e8b838b97950cb8daaeef724acbe4e3c5f/src/libraries/System.Private.CoreLib/src/System/Numerics/Hashing/HashHelpers.cs#L7-L11)
[https://github.com/dotnet/runtime/blob/327801e8b838b97950cb8daaeef724acbe4e3c5f/src/libraries/System.Private.CoreLib/src/System/HashCode.cs#L51-L54](https://github.com/dotnet/runtime/blob/327801e8b838b97950cb8daaeef724acbe4e3c5f/src/libraries/System.Private.CoreLib/src/System/HashCode.cs#L51-L54)

(This doesn't need to be addressed in this PR, but potentially work for the future)

Reply:
For any issues related to code that was moved over please file a follow up issue.  We don't want these PRs to be addressing code issues, only infra / build.

Reply:
https://github.com/dotnet/runtime/issues/33259